packages/cohere/src/cohere-chat-language-model.test.ts

```diff
@@ -68,6 +72,50 @@ describe('doGenerate', () => {
     expect(text).toStrictEqual('Hello, World!');
   });
 
+  it('should extract tool calls', async () => {
+    prepareJsonResponse({
+      text: 'Hello, World!',
+      tool_calls: [
+        {
+          name: 'test-tool',
+          parameters: { value: 'example value' },
+        },
+      ],
+    });
+
+    const { text, toolCalls, finishReason } = await model.doGenerate({
+      inputFormat: 'prompt',
+      mode: {
+        type: 'regular',
+        tools: [
+          {
+            type: 'function',
+            name: 'test-tool',
+            parameters: {
+              type: 'object',
+              properties: { value: { type: 'string' } },
+              required: ['value'],
+              additionalProperties: false,
+              $schema: 'http://json-schema.org/draft-07/schema#',
+            },
+          },
+        ],
+      },
+      prompt: TEST_PROMPT,
+    });
+
+    expect(toolCalls).toStrictEqual([
+      expect.objectContaining({
```

Comment:
why is `objectContaining` needed here?

Reply:
removed since we pass `generateId` to make the tests deterministic